src/js/tabs/onboard_logging.js

```diff
@@ -459,9 +464,12 @@ TABS.onboard_logging.initialize = function (callback) {
 
             fileEntry.createWriter(function (fileWriter) {
                 fileWriter.onerror = function (e) {
+                    GUI.log('<strong><span class="message-negative">' + i18n.getMessage('errorTitle') + ': ' + e.target.error.message + '</span class="message-negative></strong>'); 
+                        
```

Comment:
I'm not too sure about that. Is a good idea not to have tags in the messages file, but some languages are right-to-left, or use different formatting symbols (I don't know if chinese or arab for example uses `:` ).
So I think is better to put the entire string in the messages file (using a variable for the message) and let the translators give the correct form.